package.json

```diff
@@ -62,6 +62,7 @@
     "eslint": "^2.8.0",
     "eslint-plugin-babel": "^3.2.0",
     "eslint-plugin-react": "^5.1.1",
+    "glob": "^7.0.5",
```

Comment:
This should be in the normal deps as it's required to run tests. We already have `^7.0.3` there, can we bump that instead?